### PR TITLE
Update packages and switch to Yarn

### PR DESCRIPTION
**Pull request by diracdeltas**
Needed for https://github.com/brave/bitgo-proxy/issues/22

### CODE REVIEW

File: README.md
Comment (marshall) [inline README.md:8]: FWIW, we use Node v10 LTS in ethereum-remote-client via `nvm use 10` which is slightly less typing :)

**Review by marshall — APPROVED** [review]

**Comment by marshall** [comment]
@diracdeltas since I don't have write access, you'll need to get another reviewer to merge it looks like. I noticed you were using `yarn audit --json` at some point too, what happened with that code?

**Comment by diracdeltas** [comment]
@marshall that's in https://github.com/brave/bitgo-proxy/pull/23, also tagged you for review :).

**Pull request merged**
